h/static/scripts/streamsearch.coffee

```diff
@@ -1,13 +1,13 @@
 class StreamSearchController
   this.inject = [
     '$scope', '$rootScope', '$routeParams',
-    'annotator', 'auth', 'queryparser', 'searchfilter', 'store',
-    'streamer', 'streamfilter'
+    'auth', 'queryparser', 'searchfilter', 'store',
+    'streamer', 'streamfilter', 'annotationMapper'
   ]
   constructor: (
      $scope,   $rootScope,   $routeParams
-     annotator,   auth,   queryparser,   searchfilter,   store,
-     streamer,   streamfilter
+     auth,   queryparser,   searchfilter,   store,
+     streamer,   streamfilter, annotationMapper
```

Comment:
You've replaced `annotator` with `annotationMapper`, but haven't updated the code below. Also, see earlier comments about injecting both `store` and `annotationMapper`.